dpdispatcher/machines/pbs.py

```diff
@@ -17,6 +17,9 @@
 
 
 class PBS(Machine):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
Currently, I don't know why this method needs to be added. I tried to remove this method in 17eadbd55def4dc9d29776694e62c1153b2acc30, and the CI can pass as expected.